avocado/utils/software_manager.py

```diff
@@ -867,6 +867,22 @@ def provides(self, name):
         except process.CmdError:
             return None
 
+    def build_dep(self, name):
+        """
+        Install build-dependencies for package [name]
```

Comment:
Could you please add a docstring in PEP-0257 style?

https://www.python.org/dev/peps/pep-0257/

Reply:
@beraldoleal Pushed with changes, Thanks.